src/extensions/internal/extensionsconfiguration.cpp

```diff
@@ -53,7 +53,7 @@ void ExtensionsConfiguration::init()
 {
     settings()->setDefaultValue(CHECK_FOR_UPDATE, Val(true));
 
-    settings()->setDefaultValue(USER_EXTENSIONS_PATH, Val(globalConfiguration()->sharePath().toStdString() + EXTENSIONS_DIR.toStdString()));
+    settings()->setDefaultValue(USER_EXTENSIONS_PATH, Val((globalConfiguration()->appDataPath() + EXTENSIONS_DIR).toStdString()));
```

Comment:
@Eism It looks like there is a mistake. The key is called `user...` and the application data folder is used (not the user data folder)

Reply:
I think it should indeed be "userDataPath", and also a `+ "/" ` is missing.